src/components/InstanceFilters/instanceFilterRenderer.test.js

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+import { BrowserRouter as Router } from 'react-router-dom';
+
+import { screen } from '@testing-library/dom';
+import { noop } from 'lodash';
+
+import '../../../test/jest/__mock__';
+import renderWithIntl from '../../../test/jest/helpers/renderWithIntl';
+import translationsProperties from '../../../test/jest/helpers/translationsProperties';
+
+import instanceFilterRenderer from './instanceFilterRenderer';
+
+const DATA = {
+  locations: [],
+  instanceFormats: [],
+  modesOfIssuance: [],
+  natureOfContentTerms: [],
+  statisticalCodes: [],
+  query: [],
+  onFetchFacets: [],
+  parentResources: { facets: { records: [] } },
+  instanceStatuses: [],
+};
+
+const renderFilters = (data = DATA, onChange = noop) => (renderWithIntl(
+  <Router>{instanceFilterRenderer(data)(onChange)}</Router>,
+  translationsProperties
+));
+
+describe('instanceFilterRenderer function', () => {
+  beforeEach(() => renderFilters());
+  it('should display filter by tags accordion', () => {
+    expect(screen.getByText('Tags')).toBeInTheDocument();
+  });
+
+  it('Effective Location', () => {
+    expect(screen.getByText('Effective location (item)')).toBeInTheDocument();
+  });
+
+  it('effectiveLocation field', () => {
+    expect(screen.getByText('effectiveLocation-field')).toBeInTheDocument();
+  });
+
+  it('language-field', () => {
+    expect(screen.getByText('language-field')).toBeInTheDocument();
+  });
```

Comment:
These are assertions on the internals of `<InstanceFilters>`. Instead, perhaps that should be mocked so you can assert that it receives the expected props.

Reply:
Hi @zburke , We have mentioned what was present in DOM for writing test cases, could you please elaborate what needs to be mocked in this case so that we can solve the issue ASAP. 

Thanks and Regards,
Charan

Reply:
Hi @zburke,
File has been updated as suggested.

Thanks and Regards,
Charan